### PR TITLE
Repo field of login dialog is case sensitive #812

### LINKED ISSUE #812: Repo field of login dialog is case sensitive
1. Open `TEAMMATES/repo` in HubTurbo
2. Logout from HubTurbo (File -> Logout)
3. Start HubTurbo again.
4. Specify `teammates/repo` in the login dialog

In step 4, HubTurbo will attempt to redownload teammates/repo even though `TEAMMATES-repo.json` already exists in store.

### PR DESCRIPTION
**Pull request by acruis**
Fixes #812

### CODE REVIEW

**Comment by lejolly** [comment]
looks good.

**Comment by damithc** [comment]
Any chance of a test case to cover this? In general, any bug fix should come with some changes to tests.

**Comment by acruis** [comment]
Difficult. We haven't found a way to re-summon the login screen after clicking on Logout yet. (in the steps to reproduce #812, step 3 requires you to restart HubTurbo)

**Comment by acruis** [comment]
OK, added test case.

File: src/test/java/tests/StoreTests.java [hunk 1]
```diff
@@ -79,9 +79,12 @@ public void testStore() throws ExecutionException, InterruptedException {
 
         // But since we are indeed loading from the test JSON store, we would end up with 11 issues.
         Model dummy2 = alternateIO.openRepository("dummy1/dummy1").get();
-        TestUtils.delay(1); // allow for file to be written
```
Comment (lejolly) [inline src/test/java/tests/StoreTests.java]: delay no longer needed?

**Comment by lejolly** [comment]
ok looks good.

**Pull request merged**